roles/rac-db-setup/tasks/rac-db-install.yml

```diff
@@ -119,7 +119,7 @@
     regexp: "^CV_ASSUME_DISTID"
     line: "CV_ASSUME_DISTID=OEL7.8"
   when:
-    - ansible_distribution == 'OracleLinux'
+    - ansible_os_family == 'RedHat'
```

Comment:
Pardon me, I am not entirely clear on the reason for this change.

Reply:
The CV_ASSUME_DISTID change is required to install 19.3 on any RedHat-like operating system, not just Oracle Linux (MOS note 2878100.1).

Reply:
Thanks, lgtm